<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiskInit</class>
 <widget class="QDialog" name="DiskInit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>260</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>260</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>260</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Create Disk</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="infoLabel">
     <property name="text">
      <string>No disk was found. Creating a new one...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout2">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="plattersLabel">
         <property name="toolTip">
          <string>Number of Plates</string>
         </property>
         <property name="text">
          <string>Number of Platters: </string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="platterSpinBox">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="wrapping">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::PlusMinus</enum>
         </property>
         <property name="specialValueText">
          <string>default</string>
         </property>
         <property name="accelerated">
          <bool>false</bool>
         </property>
         <property name="showGroupSeparator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>4</number>
         </property>
         <property name="maximum">
          <number>12</number>
         </property>
         <property name="singleStep">
          <number>4</number>
         </property>
         <property name="value">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="tracksLabel">
         <property name="toolTip">
          <string>Number of Tracks per Plate</string>
         </property>
         <property name="text">
          <string>Number of Tracks: </string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="trackSpinBox">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::PlusMinus</enum>
         </property>
         <property name="specialValueText">
          <string>default</string>
         </property>
         <property name="minimum">
          <number>8</number>
         </property>
         <property name="maximum">
          <number>24</number>
         </property>
         <property name="singleStep">
          <number>8</number>
         </property>
         <property name="value">
          <number>8</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="sectorsLabel">
         <property name="toolTip">
          <string>Number of Sectors per Track</string>
         </property>
         <property name="text">
          <string>Number of Sectors: </string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="sectorSpinBox">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::PlusMinus</enum>
         </property>
         <property name="specialValueText">
          <string>default</string>
         </property>
         <property name="minimum">
          <number>16</number>
         </property>
         <property name="maximum">
          <number>48</number>
         </property>
         <property name="singleStep">
          <number>16</number>
         </property>
         <property name="value">
          <number>16</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="sectorSizeLabel">
         <property name="text">
          <string>Sector Size (bytes): </string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="sectorSizeSpinBox">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="specialValueText">
          <string>default</string>
         </property>
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="suffix">
          <string>b</string>
         </property>
         <property name="minimum">
          <number>512</number>
         </property>
         <property name="maximum">
          <number>4096</number>
         </property>
         <property name="singleStep">
          <number>512</number>
         </property>
         <property name="displayIntegerBase">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="blockSizeLabel">
         <property name="text">
          <string>Block Size (bytes): </string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="nameLabel">
         <property name="text">
          <string>Disk Name: </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="nameLineEdit">
         <property name="text">
          <string>disk</string>
         </property>
         <property name="maxLength">
          <number>50</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="blockSizeComboBox">
         <property name="insertPolicy">
          <enum>QComboBox::InsertAtBottom</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="modelColumn">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>512</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1024</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2048</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4096</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8192</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16384</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>32768</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>65536</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>131072</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
       <property name="centerButtons">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DiskInit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DiskInit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
